use_cases/eluc/prescriptors/prescriptor_manager.py

```diff
@@ -27,23 +30,10 @@ def prescribe(self, cand_id: str, context_df: pd.DataFrame) -> pd.DataFrame:
     def predict_metrics(self, context_actions_df: pd.DataFrame) -> pd.DataFrame:
         """
         Computes ELUC and change for each sample in a context_actions_df.
+        Returns a single dataframe with a column for each outcome
         """
-        eluc_df = self.predictor.predict(context_actions_df)
-        change_df = self.compute_percent_changed(context_actions_df)
-
-        return eluc_df, change_df
+        outcome_df = pd.DataFrame()
+        for outcome, predictor in self.predictors.items():
+            outcome_df[outcome] = predictor.predict(context_actions_df)
```

Comment:
Now we can just iterate over each outcome and predict for them. This has to be tested more in the future.